ethereum/referencetests/build.gradle

```diff
@@ -159,12 +159,12 @@ task referenceTests(type: Test) {
   classpath = sourceSets.referenceTest.runtimeClasspath
 }
 
-task ('validateReferenceTestSubmodule') {
+tasks.register('validateReferenceTestSubmodule') {
   description = "Checks that the reference tests submodule is not accidentally changed"
   doLast {
     def result = new ByteArrayOutputStream()
-    def expectedHash = 'a380655e5ffab1a5ea0f4d860224bdb19013f06a'
-    def submodulePath = java.nio.file.Path.of("${rootProject.projectDir}", "ethereum/referencetests/src/test/resources").toAbsolutePath()
+    def expectedHash = '04f338a6e673a6b4d906ec9d6d2bc939309357a5'
+    def submodulePath = java.nio.file.Path.of("${rootProject.projectDir}", "ethereum/referencetests/src/reference-test/external-resources").toAbsolutePath()
```

Comment:
What do this change do?

Reply:
Fixes the "you have the wrong submodule checked out" check.